### PR TITLE
Update i18n labels and add missing translations

### PR DESCRIPTION
**Pull request by crijke**

### CODE REVIEW

**Comment by crijke** [comment]
Moved the actual migration of the CSV export to a new pull request: 

https://github.com/finnlabs/openproject_costs/pull/21

**Comment by ulferts** [comment]
I took the liberty to rebase this branch on the current feature/rails3 branch as a merge would not have been possible otherwise.

**Comment by ulferts** [comment]
Rebased again to reflect the changes in feature/rails3. As #21 was merged to feature/rails3, commits that were also present here could be omitted. Fixed some minor issues along the way.

File: app/controllers/cost_types_controller.rb [hunk 1]
```diff
@@ -116,6 +116,6 @@ def find_cost_type
   end
 
   def default_breadcrumb
-    l(:caption_cost_type_plural)
+    CostType.model_name.human.pluralize
```
Comment (ulferts) [inline app/controllers/cost_types_controller.rb]: using pluralize will only work for english, not for other languages

**Comment by ulferts** [comment]
Rebased - again ...

**Pull request merged**